lib/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react';
+import { useStyles } from 'sku/treat';
+import { Box } from '../Box/Box';
+import { Text } from '../Text/Text';
+import { ClearButton } from '../iconButtons/ClearButton/ClearButton';
+import * as styleRefs from './Tag.treat';
+
+export interface TagProps {
+  onClear?: () => void;
+  clearLabel?: string;
+  children: ReactNode;
+}
+
+export const Tag = ({ onClear, clearLabel = 'Clear', children }: TagProps) => {
+  const styles = useStyles(styleRefs);
+
+  return (
+    <Box display="inlineBlock">
+      <Box
+        display="flex"
+        alignItems="center"
+        background="neutralLight"
+        paddingX="small"
+        paddingY={styles.constants.paddingY}
+        paddingRight={onClear ? 'xxsmall' : undefined}
+        className={styles.borderRadius}
+      >
+        <Text size={styles.constants.textSize} baseline={false}>
```

Comment:
Should we `truncate` this content? Might need to have a play and see how it would handle long content

Reply:
IMO we shouldn't force that on consumers. They should be able to trust that we'll render what they give us.

Reply:
Okay, you talked me into it. PR updated.

Reply:
What convinced you exactly @markdalgleish? Am curious.

Reply:
Given they're almost always going to be a set, having any of the tags span multiple lines will only serve to break the layout. To make this more palatable, I made sure to add a title tag with the full tag content.